### PR TITLE
Update main layout - button and font

### PR DESCRIPTION
**Pull request by monavorschulze**

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -9,27 +9,30 @@ import Ads from './components/Ads/Ads';
...
-          </>
-        ) : (
-          <Feed onMatchClick={setTargetMatchId} />
-        )}
-      </main>
-      <aside className={classes.aside}>
-        <User />
-        <Ads />
-      </aside>
+    <div>
```
Comment (AlexisRoe) [inline src/app/App.tsx:12]: 😉 the div again

File: src/app/globals.css [hunk 2]
```diff
@@ -42,12 +43,13 @@ a {
 }
 
 button {
-  background-color: var(--color-primary);
-  color: var(--color-shade-first);
+  background: linear-gradient(70deg, #fe804b, #fb31b3);
+  color: var(---color-secondary);
```
Comment (AlexisRoe) [inline src/app/globals.css:47]: small typo
```suggestion
  color: var(--color-secondary);
```

**Review by AlexisRoe — CHANGES_REQUESTED** [review]
gj

**Review by webdevelukas — APPROVED** [review]
lgtm :) 👍

File: src/app/App.tsx (hunk 1 above)
Comment (monavorschulze) [reply src/app/App.tsx:12]: arr .. it haunts me 😂 thx👍

**Pull request closed without merging**